.meta/sinks/gcp_cloud_storage.toml

```diff
@@ -0,0 +1,139 @@
+[sinks.gcp_cloud_storage]
+title = "GCP Cloud Storage (GCS)"
+beta = true
+common = true
+delivery_guarantee = "best_effort"
```

Comment:
I'm curious why this is `best_effort`? We retry until we receive a successful response, right?

Reply:
I think that's a typo. I must have copied that from somewhere else.